### PR TITLE
Refine azure migration code

### PR DESCRIPTION
**Pull request by fenxu**
1. Split includes file and normal file list.
2. Extract class for azure video helper

### CODE REVIEW

File: src/Microsoft.DocAsCode.AzureMarkdownRewriters/AzureMarked.cs [hunk 1]
```diff
@@ -26,17 +26,17 @@ public class AzureMarked
                 context = engine.Context.CreateContext(engine.Context.Variables.SetItem("path", path));
             }
 
-            if (azureMarkdownFileInfoMapping != null)
+            if (azureMarkdownFileInfoMapping?.Count != 0)
```
Comment (ansyral) [inline src/Microsoft.DocAsCode.AzureMarkdownRewriters/AzureMarked.cs]: >azureMarkdownFileInfoMapping [](start = 16, length = 28)

what if azureMarkdownFileInfoMapping is null?

Comment (fenxu) [inline src/Microsoft.DocAsCode.AzureMarkdownRewriters/AzureMarked.cs]: Just not set the video info in the context and will skip the video transform.

---
In reply to: [73485155](https://github.com/dotnet/docfx/pull/564#discussion_r73485155) [](ancestors = 73485155)

File: tools/AzureMarkdownRewriterTool/AzureFileInformationCollection.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    public class AzureFileInformationCollection
+    {
+        public AzureFileInformationCollection()
+        {
+            AzureMarkdownFileInfoMapping = new Dictionary<string, AzureFileInfo>();
+            AzureResourceFileInfoMapping = new Dictionary<string, AzureFileInfo>();
+            AzureIncludeMarkdownFileInfoMapping = new Dictionary<string, AzureFileInfo>();
+            AzureIncludeResourceFileInfoMapping = new Dictionary<string, AzureFileInfo>();
+            AzureVideoInfoMapping = new Dictionary<string, AzureVideoInfo>();
+        }
```
Comment (vwxyzh) [inline tools/AzureMarkdownRewriterTool/AzureFileInformationCollection.cs]: use property initializer

**Comment by vwxyzh** [comment]
:shipit:

**Pull request merged**